src/services/MovieService.js

```diff
@@ -5,8 +5,8 @@ const ENDPOINTS = {
 };
 
 class MovieService extends ApiService {
-  getMovies = (page) => {
-    return this.apiClient.get(ENDPOINTS.MOVIES + '?page=' + page);
+  getMovies = (payload) => {
+    return this.apiClient.get(ENDPOINTS.MOVIES + '?page=' + payload.page + "&search=" + payload.search.trim().replace(' ', '+'));
```

Comment:
ovo za mjenjanje razmaka sa + ne vjerujem da ce da radi u slucaju da pretrazijuze film `Get Out`, mozes probati opet ali mislim da ti nece vracati rezultat nazad. Takodje ovaj trim radi prilikom poziva funkcije za dovlacenje, jer neces da ti se poziva request ukoliko korinik ukuca `       ` ili ` ` sobizrom da su oba praznja mjesta i ovo si vec dovukao prethodno

Reply:
<img width="1068" alt="Screen Shot 2021-01-19 at 12 30 02" src="https://user-images.githubusercontent.com/44477868/105029050-1e790d80-5a52-11eb-9c83-ce688f162478.png">

U sustini to sa + radi, isprobao sam ja to sve bio. Sustinski mi to nije ni neophodno, tako da mogu da uklonim to.
A ovo za trim cu onda prebaciti pre okidanja akcije pa ukoliko se sadrzaj nije promenio nakon trimovanja (a to znaci da je neko kucao ` `) onda necu ni okidati akciju.